### PR TITLE
BONNIE-1150: fix duped err msg; add unit test to cover

### PR DESCRIPTION
**Pull request by lukoktonos**
This PR is to fix a duplicated Error message and to add a unit test to cover this exception.

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1150
- [x] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. NA
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [X] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA test links: NA
- [X] Justification for using JIRA tests: NA
- [X] JIRA tests have been added to sprint NA

**Reviewer 1:**

Name: @c-monkey
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass - N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree - N/A

**Reviewer 2:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass - N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree - N/A

### CODE REVIEW

**Review by c-monkey — APPROVED** [review]
Tests pass in bonnie_bundler

Doesn't pass bonnie, 'cuz we haven't merged in the JSON support yet.

looks good to me

File: test/unit/value_set_loading_test.rb [hunk 1]
```diff
@@ -66,6 +66,18 @@ class ValueSetLoadingTest < ActiveSupport::TestCase
...
+
+  test 'VSAC Error Exception Handling' do
+    # Expects that default profile will be used
+    VCR.use_cassette("vs_loading_no_profile_version") do
+      mat_file = File.new File.join("test", "fixtures", "vs_loading", "DocofMeds_v5_1_Artifacts.zip")
+      measure_details = {}
+      exception = assert_raise Measures::VSACException do
+        measure = Measures::CqlLoader.load(mat_file, "fake user", measure_details, ENV['VSAC_USERNAME'], ENV['VSAC_PASSWORD'], false, false, false)
+      end
+      assert_equal 'Error Loading Value Sets from VSAC: undefined method `id\' for "fake user":String', exception.message
```
Comment (hossenlopp) [inline test/unit/value_set_loading_test.rb:78]: I can't say this test is characteristic of an error that will ever happen, but it is effective. I tested simply giving Bonnie incorrect credentials for VSAC and the error message looked good.

screen shot 2017-10-30 at 8 01 10 am

File: lib/measures/loading/value_set_loader.rb [hunk 2]
```diff
@@ -119,7 +119,7 @@ def self.load_value_sets_from_vsac(value_sets, username, password, user=nil, ove
           delete_existing_vs(user, value_set_oids)
           backup_vs.each {|vs| HealthDataStandards::SVS::ValueSet.new(vs.attributes).save }
         end
-        raise VSACException.new "Error Loading Value Sets from VSAC: #{e.message}"
+        raise VSACException.new "#{e.message}"
```
Comment (hossenlopp) [inline lib/measures/loading/value_set_loader.rb:122]: Looking above on 113. It looks as if a bare string is raised when a value set isn't found. Wouldn't this have issues when the `e.message` is attempted?

Comment (hossenlopp) [reply lib/measures/loading/value_set_loader.rb:122]: It looks like raising a bare string actually raises a RuntimeError with the string set as the message. So this is still good.

**Review by hossenlopp — APPROVED** [review]

**Comment by hossenlopp** [comment]
Bonnie unit tests in `cql4bonnie_use_mat_artifacts` pass with this code.
